### PR TITLE
Messageboxes

### PR DESCRIPTION
**Pull request by wheybags**
Makes messageboxes during dialogue show up white in the dialogue window like they should, and while I was at it, I noticed that using the removeitem script function doesn't cause a messagebox like it should, so i did that too.

### CODE REVIEW

File: apps/openmw/mwscript/containerextensions.cpp [hunk 1]
```diff
@@ -106,12 +107,41 @@
...
+                                case MWWorld::ContainerStore::Type_Lockpick:
+                                    itemName = iter->get<ESM::Tool>()->mBase->mName; break;
+                                case MWWorld::ContainerStore::Type_Miscellaneous:
+                                    itemName = iter->get<ESM::Miscellaneous>()->mBase->mName; break;
+                                case MWWorld::ContainerStore::Type_Probe:
+                                    itemName = iter->get<ESM::Probe>()->mBase->mName; break;
+                                case MWWorld::ContainerStore::Type_Repair:
+                                    itemName = iter->get<ESM::Repair>()->mBase->mName; break;
+                                case MWWorld::ContainerStore::Type_Weapon:
+                                    itemName = iter->get<ESM::Weapon>()->mBase->mName; break;
```
Comment (ghost) [inline apps/openmw/mwscript/containerextensions.cpp]: This block of code is redundant, please use MWWorld::Class::get(iter).getName(iter)

Comment (wheybags) [inline apps/openmw/mwscript/containerextensions.cpp]: Ah, thanks, I was looking for something like that.

Comment (wheybags) [inline apps/openmw/mwscript/containerextensions.cpp]: Ah, there is a problem with that.
iter isn't an MWWorld::Ptr, it's an object with -> overloaded so it behave the same way, but it won't be accepted as a function argument.

Comment (ghost) [inline apps/openmw/mwscript/containerextensions.cpp]: Ah, right.   MWWorld::Class::get(_iter).getName(_iter) it is, then. (iter is an iterator that points to an MWWorld::Ptr)

Comment (wheybags) [inline apps/openmw/mwscript/containerextensions.cpp]: cool, thanks

**Comment by zinnschlag** [comment]
You are using a piece of literal English text in there. That can't be right. Would break any localisation. There is probably a string GMST for that somewhere.

Also, when passing strings around as argument, we usually do it as const reference.

**Pull request merged**